README.md

```diff
@@ -15,7 +20,39 @@ Website created for assessment of development techniques
 ### Live demo
 - <a href="https://joaresmiranda.github.io/2019-surfshop-poc/front-end/">Live demo</a>
 
-### Screenshots
+## How to install
```

Comment:
Please update the install  command. Now all the files live on the front-end folder